v03_pipeline/lib/misc/family_entries.py

```diff
@@ -97,26 +97,23 @@ def remove_new_callset_family_guids(
     ht: hl.Table,
     family_guids: list[str],
 ) -> hl.Table:
+    family_guids = hl.set(family_guids)
```

Comment:
The `family_guids` param comes from a `.collect()` call [here](https://github.com/broadinstitute/seqr-loading-pipelines/blob/19875488b36caae6d8ff019fdabd9fb0ef2389eb/v03_pipeline/lib/tasks/update_project_table.py#L109) - would it improve performance to keep it in hail instead of turning the guids into a python array then to a hail set?

Reply:
Yep good catch! Keeping it hail makes sense I think.  I refactored one of the internal methods to be hail instead of vanilla python so should preserve the hail everywhere.

Reply:
I just found [key_set](https://hail.is/docs/0.2/hail.expr.DictExpression.html#hail.expr.DictExpression.key_set) which is pretty much exactly what we want.